drivers/storage/portworx/manifest/manifest.go

```diff
@@ -31,6 +31,11 @@ const (
 	defaultNodeWiperImage  = "portworx/px-node-wiper:2.5.0"
 	defaultTelemetryImage  = "purestorage/ccm-service:3.0.3"
 
+	// DefaultEnvoyImage is the default Envoy image for metrics collector pod.
+	DefaultEnvoyImage = "envoyproxy/envoy:v1.19.1"
+	// DefaultCollectorImage is the default metrics collector container image.
+	DefaultCollectorImage = "pureezhang/realtime-metrics:latest"
```

Comment:
this will get overwritten via versions in manifect, which is not included in this PR. we have not decided the final image name yet, will send another PR once it is determined.